pkg/core/chain/chain.go

```diff
@@ -581,16 +581,20 @@ func (c *Chain) handleCertificateMessage(cMsg certMsg) {
 	// Fetch new intermediate block and corresponding certificate
 	cm, err := c.candidateStore.fetchCandidateMessage(cMsg.hash)
 	if err != nil {
```

Comment:
we should use a specific error to differentiate an expected state in catching the block (timeout, etc) from other exceptional situations. In the latter case we should at least log what happened